src/restApiInstance.ts

```diff
@@ -152,6 +153,10 @@ function logRequest(server: Server, request: RequestInterceptorConfig, requestId
   const config = getConfig();
   const maskedRequest = config.disableLogMasking ? request : maskRequest(request);
   const url = new URL(maskedRequest.url ?? '', maskedRequest.baseUrl);
+  if (request.params && Object.keys(request.params).length > 0) {
+    url.search = new URLSearchParams(request.params).toString();
```

Comment:
@renoyjohnm I think you asked a while back about the request query params. I was debugging something with some of the new REST APIs that require query params and the fact they were missing from the logs was making it difficult.